tests/test_snmpserver.rb

```diff
@@ -312,9 +312,9 @@ def test_snmpserver_global_enforce_priv_get_enabled
     snmpserver.global_enforce_priv = true
     device_enabled = snmpserver.global_enforce_priv?
     if device_enabled
-      refute_show_match(pattern: /no snmp-server globalEnforcePriv/)
+      assert_show_match(pattern: /snmp-server globalEnforcePriv/)
     else
-      assert_show_match(pattern: /no snmp-server globalEnforcePriv/)
+      refute_show_match(pattern: /no snmp-server globalEnforcePriv/)
```

Comment:
For that matter, what's with this test? We just explicitly set it to true at line 312, so why do we suddenly have doubts about what the current state is?